### ISSUE TITLE
Incorrect `inconsistent file name` error reported on case insensitive file systems

### ISSUE THREAD

**Issue by vladima** [Microsoft/TypeScript#5691]
testcase
```ts
        it("should not fail when module names in 'require' calls has consistent casing and current directory has uppercase chars", () => {
            const files: Map<string> = {
                "/a/B/c/moduleA.ts": `import a = require("./moduleC")`,
                "/a/B/c/moduleB.ts": `import a = require("./moduleC")`,
                "/a/B/c/moduleC.ts": "export var x",
                "/a/B/c/moduleD.ts": `
import a = require("./moduleA.ts");
import b = require("./moduleB.ts");
                `
            };
            test(files, { module: ts.ModuleKind.CommonJS, forceConsistentCasingInFileNames: true },  "/a/B/c", /* useCaseSensitiveFileNames */ false, ["moduleD.ts"], []);
        })
```

*Expected*: no errors
*Actual*: "inconsistent file name" error is reported
